cmake/tools.cmake

```diff
@@ -117,3 +117,73 @@ function(project_build_tools_get_imported_location OUTPUT_VAR_NAME TARGET_NAME)
         PARENT_SCOPE)
   endif()
 endfunction()
+
+function(project_build_tools_patch_default_imported_config)
+  set(PATCH_VARS
+      IMPORTED_IMPLIB
+      IMPORTED_LIBNAME
```

Comment:
Is there a way to know the list of patched variables here is comprehensive?

Reply:
I find them in document here https://cmake.org/cmake/help/latest/manual/cmake-properties.7.html#properties-on-targets .